GameDefinition/Content/ItemKindActor.hs

```diff
@@ -320,7 +319,7 @@ gilaMonster = ItemKind
                , AddAbility AbAlter (-2) ]  -- can't use stairs nor doors
   , ieffects = []
   , ifeature = [Durable, Identified]
-  , idesc    = ""
+  , idesc    = "Slow-moving, but with a venomous bite."
```

Comment:
The key thing about rattlesnakes is the rattle - otherwise, there's nothing special about them compared to any other snake. They definitely need something to set them apart/warn the player.